### ISSUE TITLE
I can't connect(setToken)

### ISSUE THREAD

**Issue by furkanerkl** [centrifugal/centrifuge-dart#68]
ErrorEvent{error: Connect error: ClientDisconnectedError}

 I am getting this error. I think the problem is that I can't assign the token properly. How can I use the setToken command like in JS. Also how can I make 'debug:true' as in the code below.

const centrifuge = new Centrifuge('url', {
  websocket: WebSocket,
  debug: true
});

**Comment by enesuludg** [comment]
I have same problem

**Comment by FZambia** [comment]
Hello

Could you share server and Dart SDK versions? Note that:

```
The latest centrifuge-dart is compatible only with the latest [Centrifugo](https://github.com/centrifugal/centrifugo) server (v4) and [Centrifuge](https://github.com/centrifugal/centrifuge) >= 0.25.0. For Centrifugo v2, Centrifugo v3 and Centrifuge < 0.25.0 you should use centrifuge-dart v0.8.0.
```

`setToken` method not available in the latest Dart SDK,  to work with tokens you need to use `token` and `getToken` configuration options - see [example](https://github.com/centrifugal/centrifuge-dart/blob/master/example/console/example.dart#L20). This SDK works according to [this spec](https://centrifugal.dev/docs/transports/client_api).

**Comment by furkanerkl** [comment]
@FZambia As you said, the problem was solved when I changed the library version to v0.8.0.

**Comment by FZambia** [comment]
Awesome, closing then. @enesuludg hopefully you have same issue, if not please open a new one with server/client versions and steps to reproduce
